src/main/java/pvs/app/service/GitlabApiService.java

```diff
@@ -0,0 +1,133 @@
+package pvs.app.service;
+
+import org.gitlab4j.api.*;
+import org.gitlab4j.api.models.Commit;
+import org.gitlab4j.api.models.CommitStats;
+import org.gitlab4j.api.models.Issue;
+import org.gitlab4j.api.models.Project;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import pvs.app.dto.GitlabIssueDTO;
+import pvs.app.service.thread.GitlabCommitLoaderThread;
+import pvs.app.service.thread.GitlabIssueLoaderThread;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+@Service
+public class GitlabApiService {
+
+    private final WebClient webClient;
```

Comment:
Suggest remove this variable since there only set its value but not have query & use cases in its scope.